css/style.css

```
@@ -1,19 +1,352 @@
+/* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+body {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
+
+/* Variables */
+:root {
+  --color-brand: #e0633a;
+  --color-brand-dark: #e04815;
+  --color-light: #ffffff;
+  --color-darker: #151515;
+  --color-dark: #3c3c3c;
+  --color-mid: #8d8d8d;
+  --color-cr: #dbdbdb;
+
+  --fs-h1: 4rem;
+  --fs-hero-description: 2.5rem;
+  --fs-h2: 3.5rem;
+  --fs-h3: 1.25rem;
+  --fs-navbar: 1.25rem;
+  --fs-p: 2rem;
+  --fs-obs: 1rem;
+
+  --gap-vertical: 3rem;
+
+  --ideal-page-width: 1600px;
+  --ideal-viewport-width: 1980px;
+  --vw-dif: calc(100vw - var(--ideal-viewport-width));
+  --page-width-proportion: calc(10 / 9);
+  --page-width-add: calc(var(--vw-dif) / var(--page-width-proportion));
+  --page-width: calc(var(--ideal-page-width) + var(--page-width-add));
+}
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+/* General */
+
+html {
+  background-color: var(--color-darker);
+  color: var(--color-darker);
+}
 
 body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
+  background-color: var(--color-light);
+  width: var(--page-width);
+  max-width: 100vw;
+  margin: 0 auto;
+}
+
+h1,
+h2,
+h3 {
+  font-weight: lighter;
+}
+
+h1 {
+  font-size: var(--fs-h1);
+  line-height: calc(var(--fs-h1) * 1.5);
+}
+
+h2 {
+  font-size: var(--fs-h2);
+  line-height: calc(var(--fs-h2) * 1.5);
+}
+
+h3 {
+  font-size: var(--fs-h3);
+  line-height: calc(var(--fs-h3) * 1.5);
+  font-weight: normal;
+}
+
+p {
+  font-size: var(--fs-p);
+  line-height: calc(var(--fs-p) * 1.5);
+}
+
+a {
+  color: var(--color-mid);
+  text-decoration: none;
+  transition: all 0.2s;
+}
+
+a:hover,
+a:focus {
+  color: var(--color-brand);
+  transition: all 0.2s;
+}
+
+.button {
+  margin-top: var(--gap-vertical);
+  padding: 1.25rem 2.75rem;
+  border-radius: 0.25rem;
+
+  background-color: var(--color-brand);
+  color: var(--color-light);
+  font-size: var(--fs-navbar);
+
+  transition: all 0.2s;
+}
+
+.button:hover {
+  background-color: var(--color-brand-dark);
+  color: var(--color-light);
+  transform: scale(1.05);
+  transition: all 0.2s;
+}
+
+/* Header */
+
+header {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1.25rem 8%;
+}
+
+header nav {
+  width: 50%;
+  min-width: 500px;
+  font-size: var(--fs-navbar);
+}
+
+header img {
+  height: 3rem;
+}
+
+#navbar__items {
+  display: flex;
+  justify-content: space-between;
+
+  list-style: none;
+  padding: 0;
+  margin: 0;
+
+  color: var(--color-mid);
+}
+
+.navbar__item:first-child a:not(:hover):not(:focus) {
+  color: var(--color-dark);
+}
+
+.navbar__item:first-child a {
+  font-weight: bold;
+}
+
+/* Hero */
+
+#hero {
+  background: url("../img/first-background.jpg") no-repeat top center;
```

Not very important but you can set background-size property here.
```suggestion
  background: url("../img/first-background.jpg") no-repeat top center/cover;
```